cms/admin/pageadmin.py

```diff
@@ -9,7 +9,7 @@
 import django
 from django.contrib.admin.helpers import AdminForm
 from django.conf import settings
-from django.conf.urls import url
+from django.urls import re_path
```

Comment:
I guess to ensure that the codebase worked as it did re_path is the same as url was, it is recommended to switch out to path but I think migrations like this should be staged and maybe pencil something in for a future change to use path vs re_path.